pkg/actors/table/table.go

```diff
@@ -153,26 +151,36 @@ func (t *table) Len() map[string]int {
 	return alen
 }
 
-// Drain drains actors who return true on the given func.
-func (t *table) Drain(fn func(actorType, actorID string) bool) {
+// HaltAll halts all actors currently in the table.
+func (t *table) HaltAll() error {
+	return t.Drain(func(actorType, actorID string) bool {
+		return true
+	})
+}
+
+func (t *table) Drain(fn func(actorType, actorID string) bool) error {
```

Comment:
This is super nit, but can you add back & improve the comment above this func? I think future developers could benefit from a simple comment on the diff btw `drain` vs `haltall`